app/uk/gov/hmrc/gform/formtemplate/Verifier.scala

```diff
@@ -157,6 +158,84 @@ trait Verifier {
     case fc => fc.copy(mandatory = false)
   }
 
+  def specimenSmartStrings(formTemplate: FormTemplate): FormTemplate = {
+
+    def updateSpecimenFormComponent(formComponent: FormComponent): FormComponent = {
+      def updateLabel(fc: FormComponent): FormComponent =
+        fc.copy(label = updateSpecimenSmartString(fc.label))
+
+      def updateHelpText(fc: FormComponent): FormComponent =
+        fc.copy(helpText = fc.helpText.map(updateSpecimenSmartString))
+
+      def updateInfoText(fc: FormComponent): FormComponent =
+        fc.`type` match {
+          case InformationMessage(infoType, infoText) =>
+            fc.copy(`type` = InformationMessage(infoType, updateSpecimenSmartString(infoText)))
+          case otherwise => fc
+        }
+
+      def updateChoices(fc: FormComponent): FormComponent = {
+        import OptionData._
+        fc.`type` match {
+          case c: Choice =>
+            val updatedOptions = c.options.map {
+              case IndexBased(label, includeIf)        => IndexBased(updateSpecimenSmartString(label), includeIf)
+              case ValueBased(label, value, includeIf) => ValueBased(updateSpecimenSmartString(label), value, includeIf)
+            }
+            val updatedChoiceType = c.copy(options = updatedOptions)
+            fc.copy(`type` = updatedChoiceType)
+          case otherwise => fc
+        }
+      }
+      (updateLabel _ andThen updateHelpText _ andThen updateInfoText _ andThen updateChoices)(formComponent)
```

Comment:
Why is this manipulating help text or info text or even a choices?

Reply:
the ticket enumerates the places where it should be replaced. I just implemented ACs